README.md

```diff
@@ -1,8 +1,8 @@
 BitShares Core
 ==============
 
-[BitShares Core](https://github.com/bitshares/bitshares-core) is the BitShares blockchain implementation and command-line interface.
-The web browser based wallet is [BitShares UI](https://github.com/bitshares/bitshares-ui).
+[BitShares Core](https://github.com/bitshares/bitshares-core) is the BitShares blockchain node software and command-line wallet software.
+For UI reference wallet software visit [BitShares UI](https://github.com/bitshares/bitshares-ui).
```

Comment:
I still think it is better to mention "browser-based" or "desktop app", because average users may not know what "reference wallet" means.